src/plugins/console/public/styles/_app.scss

```diff
@@ -33,6 +33,42 @@
 .conApp__output {
   display: flex;
   flex: 1 1 1px;
+
+  .ace_badge {
+    font-size: 12px;
+    font-weight: 500;
+    line-height: 18px;
+    padding: 0 8px;
+    display: inline-block;
+    text-decoration: none;
+    border-radius: 3px;
+    border: solid 1px transparent;
+    white-space: nowrap;
+    vertical-align: middle;
+    cursor: default;
+    max-width: 100%;
+    text-align: left;
```

Comment:
In dark mode, the text for this badge appears white:

![image](https://user-images.githubusercontent.com/1238659/169879706-85d06bb0-255e-4aa4-9e57-ac9c42257197.png)

We can add `color: black;` to fix this, but this seems inelegant. @elastic/eui-design Any suggestions on how we should incorporate EUI's coloring rules into these styles, to get this badge to match `EuiBadge`?

![image](https://user-images.githubusercontent.com/1238659/169879601-695340ba-35a0-4dec-9b57-8936bc941774.png)

Reply:
Via: https://elastic.github.io/eui/#/utilities/color-functions#light-or-dark-text?themeLanguage=sass

```sass
color: chooseLightOrDarkText($color);
```

Reply:
TY!